scripts/presets/travis/replica_sets/replicaset.json

```diff
@@ -0,0 +1,66 @@
+{
+    "id": "REPLICASET",
+    "name": "mongod",
+    "members": [
+        {
+            "procParams": {
+                "dbpath": "/tmp/REPLICASET/3000/",
+                "ipv6": true,
+                "logappend": true,
+                "logpath": "/tmp/REPLICASET/3000/mongod.log",
+                "journal": true,
+                "nssize": 1,
+                "port": 3000,
+                "bind_ip_all": true,
+                "smallfiles": true
```

Comment:
Per https://github.com/10gen/mongo-orchestration/pull/247#issuecomment-403333778, I think we can remove `nssize`, `smallfiles`, and `noprealloc` from all of these configs.

Only the `STANDALONE_OLD` case under the 3.0.x server version will use MMAPv1 (where `noprealloc` might still apply); however, I'm not convinced it's worth optimizing for that case (especially since it's not even a replica set). Having these servers use `/tmp` as their DB path should be enough optimization, since I would expect that to be a RAM disk.

I had previously removed `noprealloc` from our non-Travis configs in https://github.com/mongodb/mongo-php-driver/commit/2ef765acaff37872e26a767ec1db5da9d579df6a but stupidly added the others in https://github.com/mongodb/mongo-php-driver/commit/9a83bde5824deca68be8cda688133f3f235f06a1. Perhaps we can remove all three options across the board (for these Travis configs and the non-Travis configs).

Reply:
I've made a ticket for it, to do this as a low priority thing later: https://jira.mongodb.org/browse/PHPC-1240